language/German/strings.po

```diff
@@ -1980,29 +2007,34 @@ msgid ""
 "\n"
 "Press any key to continue."
 msgstr ""
-"Der gewählten Maschine fehlen ein oder mehrere benötigte ROM- oder CHD-"
-"Abbilder. Bitte wählen Sie eine andere Maschine aus.\n"
+"Der gewählten Maschine fehlen ein oder mehrere benötigte ROM- oder CHD-Abbilder. "
+"Bitte wählen Sie eine andere Maschine aus.\n"
 "\n"
 "Drücken Sie eine beliebige Taste, um fortzufahren."
 
 #: src/frontend/mame/ui/simpleselgame.cpp:278
 msgid "Configure Options"
 msgstr "Optionen konfigurieren"
 
-#: src/frontend/mame/ui/simpleselgame.cpp:279
-#: src/frontend/mame/ui/selmenu.cpp:1248
+#: src/frontend/mame/ui/simpleselgame.cpp:279 src/frontend/mame/ui/selmenu.cpp:1248
 msgid "Exit"
 msgstr "Verlassen"
 
 #: src/frontend/mame/ui/simpleselgame.cpp:304
 #, c-format
 msgid ""
-"No system ROMs found. Please check the rompath setting specified in the %1$s."
-"ini file.\n"
+"No system ROMs found. Please check the rompath setting specified in the %1$s.ini "
+"file.\n"
 "\n"
-"If this is your first time using %2$s, please see the %2$s.pdf file in the "
-"docs folder for information on setting up and using %2$s."
+"If this is your first time using %2$s, please see the %2$s.pdf file in the docs "
+"folder for information on setting up and using %2$s."
```

Comment:
Please back out this re-wrapped English message.

Reply:
Fixed